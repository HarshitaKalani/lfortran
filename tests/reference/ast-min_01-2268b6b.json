{
    "basename": "ast-min_01-2268b6b",
    "cmd": "lfortran --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/errors/min_01.f90",
    "infile_hash": "4f2b6a5d17d5444ec72785deb89d7ba6cfb6e8d5dd05afde0d4ebdc4",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-min_01-2268b6b.stdout",
    "stdout_hash": "b80b73b825b052e13e23b1633299d1403fed185fcba1c567953adb37",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}