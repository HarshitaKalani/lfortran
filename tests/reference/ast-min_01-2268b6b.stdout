(TranslationUnit
    [(Program
        min_01
        ()
        []
        []
        [(Declaration
            (AttrType
                TypeReal
                [(()
                8
                Value)]
                ()
                ()
                None
            )
            []
            [(y
            []
            []
            ()
            None
            ())]
            ()
        )
        (Declaration
            (AttrType
                TypeReal
                []
                ()
                ()
                None
            )
            []
            [(z
            []
            []
            ()
            None
            ())]
            ()
        )]
        [(=
            0
            y
            (Real "5.2d0")
            ()
        )
        (=
            0
            z
            (Real "9.0")
            ()
        )
        (Print
            0
            ()
            [(FuncCallOrArray
                min
                []
                [(()
                y
                ()
                0)
                (()
                z
                ()
                0)]
                []
                []
                []
            )]
            ()
        )]
        []
    )]
)
